### PR TITLE
adding direcctory read permissions to the aad app

### PR DESCRIPTION
**Pull request by JackQuincy**
adding aad directory permissions

### CODE REVIEW

File: src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py [hunk 1]
```diff
@@ -2254,9 +2254,14 @@ def _ensure_osa_aad(cli_ctx,
         # Delegate Sign In and Read User Profile permissions on Windows Azure Active Directory API
         resource_access = ResourceAccess(id="311a71cc-e848-46a1-bdf8-97ff7156d8e6",
                                          additional_properties=None, type="Scope")
-        required_osa_aad_access = RequiredResourceAccess(resource_access=[resource_access],
+        # Read directory permission on Windws Azure Active Directory API
```
Comment (amanohar) [inline src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py]: nit: Typo

**Review by amanohar — APPROVED** [review]

File: src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom_BACKUP_23586.py [hunk 2]
```diff
@@ -0,0 +1,2606 @@
+# --------------------------------------------------------------------------------------------
```
Comment (julienstroheker) [inline src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom_BACKUP_23586.py]: What are those files ?

File: src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py [hunk 3]
```diff
@@ -2254,9 +2254,14 @@ def _ensure_osa_aad(cli_ctx,
...
+        directory_access = ResourceAccess(id="5778995a-e1bf-45b8-affa-663a9f3f4d04",
```
Comment (julienstroheker) [inline src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py:2258]: Just curious what is this ?

Comment (amanohar) [reply src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py:2258]: @julienstroheker: "5778995a-e1bf-45b8-affa-663a9f3f4d04" -> Read directory data

**Pull request merged**